.github/workflows/ci.yml

```diff
@@ -16,10 +16,7 @@ jobs:
       - name: Install packages
         run: |
           brew update
-          brew install pkg-config ninja || brew link --overwrite python
-
-      - name: Install python modules
-        run: pip3 install meson==0.49.2
+          brew install pkg-config ninja meson
```

Comment:
This will install the latest version of `meson`. Is that OK?

Reply:
yes, it should work